### PR TITLE
V3.8.0  fix fbx and widget bug in inspector

### PR DESCRIPTION
**Pull request by arsen2010**
Re: #https://github.com/cocos/3d-tasks/issues/16896
https://github.com/cocos/3d-tasks/issues/16956

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: editor/inspector/assets/fbx/events/events.js [hunk 1]
```diff
@@ -97,7 +96,7 @@ exports.methods = {
     },
 
     queryKeyStyle(x) {
-        return `transform: translateX(${x | 0 + 3}px);`;
+        return `transform: translateX(${x | 0}px);`;
```
Comment (doommm) [inline editor/inspector/assets/fbx/events/events.js]: x | 0 ? x || 0 ?

File: editor/inspector/assets/fbx/index.js [hunk 2]
```diff
@@ -2,6 +2,8 @@
 const path = require('path');
 const { injectionStyle } = require('../../utils/prop');
 
+let cacheActiveTab = 'animation';
```
Comment (doommm) [inline editor/inspector/assets/fbx/index.js:5]: 这个是全局的缓存，会对所有的 fbx panel 生效，是否符合预期？

File: editor/inspector/assets/fbx/preview.js [hunk 3]
```diff
@@ -762,7 +769,7 @@ exports.update = async function (assetList, metaList) {
     this.refreshPreview();
 };
 
-exports.close = function () {
+exports.close = function() {
```
Comment (doommm) [inline editor/inspector/assets/fbx/preview.js:766]: 上面三处的格式要还原

File: editor/inspector/assets/fbx/index.js (hunk 2 above)
Comment (arsen2010) [reply editor/inspector/assets/fbx/index.js:5]: it is ok

File: editor/inspector/assets/fbx/events/events.js (hunk 1 above)
Comment (arsen2010) [reply editor/inspector/assets/fbx/events/events.js]: modify use x || 0

File: editor/inspector/assets/fbx/preview.js (hunk 3 above)
Comment (arsen2010) [reply editor/inspector/assets/fbx/preview.js:766]: eslint auto format

**Review by doommm — APPROVED** [review]

**Review by VisualSJ — APPROVED** [review]

**Pull request merged**